application.windows64/java/Welcome.html

```
@@ -0,0 +1,28 @@
+<html>
+<head>
+<title>
+Welcome to the Java(TM) Platform
+</title>
+</head>
+<body>
+
+<h2>Welcome to the Java<SUP><FONT SIZE=-2>TM</FONT></SUP> Platform</h2>
```

HTMLHint (1.5.0)

[error] The html element name of [ SUP ] must be in lowercase.

(tagname-lowercase)